packages/core/test/integration/query-interface.test.js

```diff
@@ -233,12 +233,19 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     });
 
     it('adds, reads and removes an index to the table', async function () {
-      await this.queryInterface.addIndex('Group', ['username', 'isAdmin']);
-      let indexes = await this.queryInterface.showIndex('Group');
+      await this.queryInterface.addIndex('Group', { fields: ['username', 'isAdmin'] });
+      let indexes = await this.queryInterface.showIndexes('Group');
       let indexColumns = uniq(indexes.map(index => index.name));
-      expect(indexColumns).to.include('group_username_is_admin');
-      await this.queryInterface.removeIndex('Group', ['username', 'isAdmin']);
-      indexes = await this.queryInterface.showIndex('Group');
+      // MariaDB & MySQL support index via alter and if no name is given, the db sets the name not sequlize
+      if (['mariadb', 'mysql'].includes(dialectName)) {
+        expect(indexColumns).to.include('username');
```

Comment:
there was a test for removing an index based on the list of columns before, I don't see it anymore

Reply:
I've added this in now.